pkg/kor/exporter.go

```diff
@@ -0,0 +1,73 @@
+package kor
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	orphanedResourcesCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "Kubernetes_Orphaned_Resources",
+			Help: "Orphaned resources in Kubernetes",
+		},
+		[]string{"kind", "namespace", "resourceName"},
+	)
+)
+
+func init() {
+	prometheus.MustRegister(orphanedResourcesCounter)
+}
+
+// TODO: add option to change port / url !?
+func Exporter(includeExcludeLists IncludeExcludeLists, clientset kubernetes.Interface, outputFormat string) {
+	http.Handle("/metrics", promhttp.Handler())
+	fmt.Println("Server listening on :8080")
+	go exportMetrics(includeExcludeLists, clientset, outputFormat) // Start exporting metrics in the background
+	if err := http.ListenAndServe(":8080", nil); err != nil {
+		fmt.Println(err)
+	}
+}
+
+func exportMetrics(includeExcludeLists IncludeExcludeLists, clientset kubernetes.Interface, outputFormat string) {
+	exporterInterval := os.Getenv("exporterInterval")
```

Comment:
Can you convert environment variable names to uppercase and separated by underscores? For example: `EXPORTER_INTERVAL`